it/mail/chrome/messenger/subscribe.dtd

```diff
@@ -35,10 +35,10 @@
 
  ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY subscribeDialog.title        "Sottoscrizioni">
-<!ENTITY subscribeButton.label        "Sottoscrivi">
+<!ENTITY subscribeDialog.title        "Iscrizioni">
+<!ENTITY subscribeButton.label        "Iscriviti">
 <!ENTITY subscribeButton.accesskey    "S">
-<!ENTITY unsubscribeButton.label      "Rimuovi sottoscrizione">
+<!ENTITY unsubscribeButton.label      "Rimuovi iscrizione">
```

Comment:
CORRECTION NEEDED. 

In order to improve the overall consistency, I would suggest to use "Disiscriviti" instead of "Rimuovi iscrizione".